libimage/save.go

```diff
@@ -56,13 +56,13 @@ func (r *Runtime) Save(ctx context.Context, names []string, format, path string,
 
 	// Dispatch the save operations.
 	switch format {
-	case "oci-archive", "oci-dir", "docker-dir":
+	case "oci-archive", "oci-dir", "docker-dir", "dir":
 		if len(names) > 1 {
 			return errors.Errorf("%q does not support saving multiple images (%v)", format, names)
 		}
 		return r.saveSingleImage(ctx, names[0], format, path, options)
 
-	case "docker-archive":
+	case "docker-archive", "compat-archive":
 		options.ManifestMIMEType = manifest.DockerV2Schema2MediaType
```

Comment:
(Completely unrelated, I don’t see why this is hard-coded here. The transport already contains only that value in `SupportedManifestMIMETypes` — so this maybe just silently ignores user’s explicit format value instead of failing.

Is there some non-obvious reason for doing this? If so, it might be worth a comment.)

Reply:
Probably just old code. I can remove it.

Reply:
I’m really after the knowledge, not after dropping the line. It came in https://github.com/containers/common/pull/551 and looks fairly unlikely to be accidental, but it’s not obvious to me why it was added.

If we don’t know, let’s leave it it, at least in _this_ PR.